expr: refresh page (#7999)
tldr-pages/tldr: pages/common/expr.md

> Evaluate expressions and manipulate strings.
> More information: <https://www.gnu.org/software/coreutils/expr>.

> - Get a specific string length:

Reviewer: How about:
```suggestion
- Get the length of a specific string:
```

Author: I personally don't like it due to `the length of`. As I understand it is preferred to use `a`/`an` articles instead of `the` to make descriptions more general. That's why I refresh many pages now to standardize them. 😄

Reviewer 2: `the length of` makes more sense in this context.

Author: I don't know why it makes more sense but it maybe more readable. 😄

> `expr length '{{hello world}}'`

Reviewer 3: ```suggestion
`expr length '{{string}}'`
```
I think the old placeholder is fine here.

Author: Yes, it's definitely better.

- Get the substring of a string with a specific length:

`expr substr "{{string}}" {{from}} {{length}}`

> - Match a specific substring agains an anchored pattern:

Reviewer 3: ```suggestion
- Match a specific substring against an anchored pattern:
```

`expr match {{string}} {{pattern}}`

- Get a substring position in a specific string:

> `expr index {{string}} {{substring}}`

Reviewer 4: This command prints the first occurrence of a char in a "collection"  of chars. This doesn't work with substring. e.g.  
```bash
$ expr index freeradius radius
2
$ expr index freeradius free
1
$ expr index freeradius dius
7
$ expr index freeradius abdf
1
```

Author: Fixed. :)

> - Calculate a specific mathematic expession:

Reviewer 3: ```suggestion
- Calculate a specific mathematic expression:
```

`expr {{expression1}} {{+|-|*|/|%}} {{expression2}}`

> - Get the first expression if it's value non-zero and not null otherwise get the second one:

Reviewer 3: ```suggestion
- Get the first expression if its value is non-zero and not null otherwise get the second one:
```
